.github/workflows/ci.yaml

```diff
@@ -0,0 +1,24 @@
+name: CI
+on:
+  push:
+    branches: [main]
+  pull_request:
+  workflow_dispatch:
+jobs:
+  integration-test-in-studio-container:
+    runs-on: ubuntu-latest
+    env:
+      USER_WS: /github/workspace
+    # TODO: docker tag
+    # TODO: example for custom base images
+    container: picknikciuser/moveit-studio:main
+    steps:
+      - uses: actions/checkout@v4
+      - name: Run colcon build and test
```

Comment:
Maybe worth separating into build and test? It's nice to always get test results, for instance, and this won't do it if the tests fail I think.


```
      - name: Run tests
        run: |
          . /opt/ros/humble/setup.sh
          . /opt/overlay_ws/install/setup.sh
          colcon test
      - if: always()
        run: colcon test-result --verbose
```

Reply:
oh yes actually I have it like that in another branch... I've been peeling off changes to get this in